modules/configuring-distributed-workloads.adoc

```diff
@@ -10,22 +10,36 @@ To configure the distributed workloads feature for your data scientists to use i
 * You have logged in to {openshift-platform} with the `cluster-admin` role.
 * You have sufficient resources. In addition to the base {productname-short} resources, you need 1.1 vCPU and 1.6 GB memory to deploy the distributed workloads infrastructure.
 * You have access to a Ray cluster image. For information about how to create a Ray cluster, see the link:https://docs.ray.io/en/latest/cluster/getting-started.html[Ray Clusters documentation].
-* You have removed any previously installed instances of the CodeFlare Operator, as described in the Knowledgebase solution link:https://access.redhat.com/solutions/7043796[How to migrate from CodeFlare Operator to Red Hat OpenShift AI Cluster].
 ifndef::upstream[]
-* If you want to use graphics processing units (GPUs), you have enabled GPU support in {productname-short}. See link:{rhoaidocshome}{default-format-url}/managing_resources/managing-cluster-resources_cluster-mgmt#enabling-gpu-support_cluster-mgmt[Enabling GPU support in {productname-short}].
+* You have removed any previously installed instances of the CodeFlare Operator, as described in the Knowledgebase solution link:https://access.redhat.com/solutions/7043796[How to migrate from CodeFlare Operator to Red Hat OpenShift AI Cluster].
+* If you want to use graphics processing units (GPUs), you have enabled GPU support in {productname-short}.
+See link:{rhoaidocshome}{default-format-url}/managing_resources/managing-cluster-resources_cluster-mgmt#enabling-gpu-support_cluster-mgmt[Enabling GPU support in {productname-short}].
```

Comment:
As a matter of interest, do Habana Gaudi devices also support distributed workloads? If so, we'll need to get something added on Habana Gaudi devices in  the prereqs. Perhaps not in this PR, but in another PR at a future release.

Reply:
This module is not the only module with that GPU prereq, so perhaps you should review all such modules in a separate PR, to see if an equivalent Habana Gaudi prereq should be added?